docs/en_US/Compression/QuickStart.md

```diff
@@ -193,4 +195,24 @@ pruner.export_model(model_path='model.pth')
 pruner.export_model(model_path='model.pth', mask_path='mask.pth', onnx_path='model.onnx', input_shape=[1, 1, 28, 28])
 ```
 
+#### Export Quantized Model
+
+You can export the quantized model directly by using ```torch.save``` api and the quantized model can be loaded by ```torch.load``` without any extra modification. The following example shows the normal proceduce of saving, loading quantized model and get related parameters in QAT.
+```
+# save quantized model
+torch.save(model.state_dict(), "quantized_model.pkt")
```

Comment:
it is not clear what the relation between `model` and `model_1` below...

Reply:
Have changed model_1 to qmodel_load and add related annotation.